### ISSUE TITLE
Slides with missing logo image slides just say logo

### ISSUE THREAD

**Issue by alapitz** [IAmParadox27/jellyfin-plugin-media-bar#123]
When content doesn't have a logo it simply display's logo. The Alt text could be updated to say the title instead making it more useful in those cases. 

All that needs to be done is update this https://github.com/IAmParadox27/jellyfin-plugin-media-bar/blob/e246a616cd3ba71df94b6af920670eb2717ce046/slideshowpure.js#L1263 to be title instead of item.name

**Comment by alapitz** [comment]
@IAmParadox27  poke, just keeping this alive.

**Comment by alapitz** [comment]
@IAmParadox27 keeping alive

**Comment by IAmParadox27** [comment]
@MakD one for you in yours I think

**Comment by timojaakob** [comment]
I have a similar issue, for some movies it just says the title in regular text instead of showing the logo, even though the logo exists and is visible upon opening the movies page in jellyfin.

**Comment by alapitz** [comment]
@IAmParadox27, Sorry for the misdirection this actually appears to be an issue with how this repo imports slideshow.js based on the jellyfin version and is already fixed in @MakD version and the version in the code. The issue is simply due to how which version of the slideshowpure.js gets imported 

When using 10.11 this repo will import https://cdn.jsdelivr.net/gh/IAmParadox27/jellyfin-plugin-media-bar@10.11/slideshowpure.js which has a static text "Logo" for the alt text instead it needs to pull in the main version https://cdn.jsdelivr.net/gh/IAmParadox27/jellyfin-plugin-media-bar@main/slideshowpure.js which correctly uses the movie name. 

I haven't had the time to figure out how the templating engine works for the index to do the conditional or if you just want to pull the main version always now since 10.11 has been out for awhile seems safe to do imo.

**Comment by IAmParadox27** [comment]
You should be able to just change the version in the plugin settings to "main" and it pull this in without a plugin change :)

**Comment by alapitz** [comment]
Oh nice! I didn't see that setting. Thanks for the help.
